en/mavgen_c/README.md

```diff
@@ -59,16 +60,164 @@ When compiling the project, we recommend that you specify the top-level output d
 $ gcc ... -I generated/include -I generated/include/common ...
 ```
 
-## Multiple Streams ("channels") {#streams}
+## Multiple Streams ("channels") {#channels}
 
-The C MAVLink library utilizes a "channel" metaphor to allow for simultaneous processing of multiple, independent MAVLink streams in the same program. 
-It is therefore important to use the correct channel for each operation as all receiving and transmitting functions provided by MAVLink require a channel.
+The C MAVLink library utilizes a "channel" metaphor to allow for simultaneous processing of multiple, independent MAVLink streams in the same program.
+All receiving and transmitting functions provided by this library require a channel, and it is important to use the correct channel for each operation.
 
+Up to 4 channels can be defined on MCU/ARM7 while up to 16 may be defined on Linux/Windows.
```

Comment:
Perhaps a "by default" and `s/can be/are/`.  ArduPilot specifies more channels.

Reply:
... also, `MCU/ARM7` is garbage, it's "everything other than xyzzy":

```
#ifndef MAVLINK_COMM_NUM_BUFFERS
#if (defined linux) | (defined __linux) | (defined  __MACH__) | (defined _WIN32)
# define MAVLINK_COMM_NUM_BUFFERS 16
#else
# define MAVLINK_COMM_NUM_BUFFERS 4
#endif
```

Poor MacOS users, always forgotten :-)

Reply:
Thanks, will fix. 

> Poor MacOS users, always forgotten :-)

Yes, because they suck :-)